TimoCloud-Universal/src/main/java/cloud/timo/TimoCloud/core/managers/APIRequestManager.java

```diff
@@ -408,7 +408,9 @@ public void onPluginMessage(AddressedPluginMessage message) {
                             String targetServer = data.getString("targetServer");
                             validateNotNull(playerUUID, "UUID");
                             validateNotNull(targetServer, "targetServer");
-                            responseData = (T) Boolean.valueOf(proxy.getOnlinePlayers().stream().anyMatch(playerObject -> playerObject.getUuid().toString().equals(playerUUID)));
+                            Boolean isOnGivenProxy = Boolean.valueOf(proxy.getOnlinePlayers().stream().anyMatch(playerObject -> playerObject.getUuid().toString().equals(playerUUID)));
+                            responseData = (T) isOnGivenProxy;
+                            if(isOnGivenProxy)
```

Comment:
Thanks for implementing the boolean response. However, you will have to change the signature of the API Method to APIRequestFuture<Boolean>. Did you test your code?

P.S.: Please use proper code style in the if-statement :)

Reply:
yes, i have tested this code.